### PR TITLE
update LLVM version to 18

### PR DESCRIPTION
**Pull request by x14ngch3n**
https://github.com/banach-space/llvm-tutor/issues/117

### CODE REVIEW

File: Dockerfile_ubuntu [hunk 1]
```diff
@@ -6,17 +6,18 @@
...
 # wget https://raw.githubusercontent.com/banach-space/llvm-tutor/main/Dockerfile_ubuntu
-# docker build -t=llvm-tutor:llvm-16 .
-# docker run --rm -it --hostname=llvm-tutor llvm-tutor:llvm-16 /bin/bash
+# docker build -t=llvm-tutor:llvm-18 .
+# docker run --rm -it --hostname=llvm-tutor llvm-tutor:llvm-18 /bin/bash
 # ```
 # =============================================================================
 
-FROM debian:buster
+FROM debian:22.04
```
Comment (banach-space) [inline Dockerfile_ubuntu:14]: Did you mean ubuntu rather than debian?

Comment (x14ngch3n) [reply Dockerfile_ubuntu:14]: Yes, sorry for the mistake

**Comment by x14ngch3n** [comment]
The docker-archlinux failed because it always downloads the code from main branch other than my PR branch, but I test it on my machine and it builds successfully

**Comment by banach-space** [comment]
Thank you for helping out - this part of **llvm-tutor** was crying for a bit of love. Great work - much appreciated!

**Pull request merged**